src/current/_plugins/sidebar_htmltest.rb

```diff
@@ -1,16 +1,36 @@
 require 'json'
 require 'liquid'
+require 'yaml'
 
 module SidebarHTMLTest
   class Generator < Jekyll::Generator
     def generate(site)
       @site = site
-
+      
+      # Read htmltest configuration to get ignored directories
+      htmltest_config = YAML.load_file('.htmltest.yml') rescue {}
+      ignored_dirs = htmltest_config['IgnoreDirs'] || []
+      
+      # Extract version numbers from ignored directories
+      ignored_versions = ignored_dirs.map do |dir|
+        match = dir.match(/\^?docs\/?(v\d+\.\d+)/)
+        match[1] if match
+      end.compact
```

Comment:
interesting, I don't think `end.compact` is valid Ruby syntax, but it does pass `ruby -c` 🤔